apps/mymedicare_cb/views.py

```diff
@@ -109,6 +109,11 @@ def callback(request):
         crosswalk.fhir_id = fhir_id
         crosswalk.save()
 
+        logger.info("Beneficiary connected to FHIR: %s" % crosswalk.fhir_id)
+    else:
```

Comment:
At this point it looks like the function should exit or raise an exception. Can the auth process continue, in any meaningful way, if there's no FHIR ID?

Reply:
In Prod we are not getting the FHIR ID written to the crosswalk. So subsequent calls fail with 500 error. 
I would like to see a success message in the logs because that could be used by Splunk to monitor growth in number of connected Beneficiaries.
I  currently suspect that the first call to the patient search is succeeding but the structure of the record may have changed so we don't get the _Id.

Reply:
In PROD we are seeing Crosswalk records created but no FHIR_ID populated. I want to run a test with Ryan today to try a curl call and see what we get back from the back-end